### DISCUSSION TITLE
Sablier Nginx routing query

### DISCUSSION THREAD

**Discussion by vutsalsinghal** [sablierapp/sablier#164] [Q&A]
Hi There,

This is a followup question to the discussion in https://github.com/acouvreur/sablier/discussions/161

Consider `whoami_container` is down currently. When I go to `who.exmaple.com`, I have achieved this workflow without issues:
```
who.exmaple.com -> who.exmaple.com/dynamic/whoami_container ->(sablier brings up container) -> who.exmaple.com
```
I'm able to achieve this by using something like this in nginx
```
error_page 502 503 = @error50x;
location @error50x {
    return 302 /dynamic/whoami_container;
}
```
But how do I achieve this?
```
who.exmaple.com/some-uri -> who.exmaple.com/dynamic/whoami_container ->(sablier brings up container) ->  who.exmaple.com/some-uri
```
I tried a few things but I was not able to redirect (or preserve) the the original uri (/some-uri) once the container was up...

**Comment by acouvreur** [comment]
My question is, why would you need the redirect in the first place?

The goal of this plugin is to intercept request in-flight. So no redirects are needed.

However, if you really need this, then I'm not so sure how it could work. Maybe you could just use the Sablier endpoint directly instead of the plugin

**Reply by vutsalsinghal** [reply]
> The goal of this plugin is to intercept request in-flight. So no redirects are needed.

could you expand a bit on this please with an examaple if possible
